cmd/juju/charmhub/info_test.go

```diff
@@ -204,19 +574,20 @@ charm:
   - wordpress-everlast
   - wordpress-jorge
   - wordpress-site
-channel-map:
-  latest/stable:
-    released-at: "2019-12-16T19:44:44.076943+00:00"
-    track: latest
-    risk: stable
-    revision: 16
-    size: 12042240
-    version: 1.0.3
-    architectures:
-    - amd64
-    bases:
-    - ubuntu/16.04
-    - ubuntu/18.04
+channels:
```

Comment:
The diff here gives you an idea of what's actually changed structurally in the YAML/JSON output.